proto/src/main/protobuf/FlightsMessage.proto

```diff
@@ -76,4 +76,10 @@ message FeedStatusMessage {
 message TotalPaxSourceMessage {
     optional int32 pax = 1;
```

Comment:
If this field is no longer being used then we can rename it by appending `__OLD` and move it down to the bottom of the list of fields. There's good documentation on updating protobuf messages here: https://protobuf.dev/programming-guides/proto2/#updating